docker/Dockerfile

```diff
@@ -1,7 +1,9 @@
 FROM grafana/grafana:5.2.2
-RUN curl -L https://github.com/telia-oss/aws-env/releases/download/v0.3.0/aws-env-linux-amd64 > /tmp/aws-env && \
-      echo f80addd4adf9aa8d4ecf1b16de402ba4  /tmp/aws-env | md5sum -c && \
-      chmod +x /tmp/aws-env
+USER root
+RUN curl -L https://github.com/telia-oss/aws-env/releases/download/v0.3.0/aws-env-linux-amd64 > /usr/local/bin/aws-env && \
+      echo f80addd4adf9aa8d4ecf1b16de402ba4  /usr/local/bin/aws-env | md5sum -c && \
```

Comment:
What is this magic string?

Reply:
This downloads aws-env and checks it against the md5 checksum (that I manually calculated)

Reply:
Ahh, makes sense 👍